queries.sql

```diff
@@ -0,0 +1,22 @@
+insert into users (username, password, role)
+  values ('Admin','$2a$10$D/IX/AtYw5YHT4YWI2B2aOg5ZGMBodHNDx2x6vIbHWomyH4fsJ9SG','admin');
+insert into users (username, password, role)
+  values ('ydrc','$2a$10$D/IX/AtYw5YHT4YWI2B2aOg5ZGMBodHNDx2x6vIbHWomyH4fsJ9SG','user');
+
+insert into events (title,start_date,end_date,org_name,price,capacity,note,"userId",status)
+  values ('Interview',DATE 'Thu, 14 February 2019 13:30:00', DATE 'Thu, 14 February 2019 13:30:00','YDRC',120,10,'The event is for the interview',0,1);
```

Comment:
Here's ```userId=0```  Ids are starting counting from 1 not from 0

Reply:
done :+1: